test/helpers/ProposalTest.sol

```diff
@@ -153,4 +159,162 @@ abstract contract ProposalTest is PooltogetherGovernorTest {
     // Ensure the new proposal is now executed
     assertEq(governorAlpha.state(_newProposalId), EXECUTED);
   }
+
+  function _buildProposalData(string memory _signature, bytes memory _calldata)
+    internal
+    pure
+    returns (bytes memory)
+  {
+    return abi.encodePacked(bytes4(keccak256(bytes(_signature))), _calldata);
+  }
+
+  function _jumpToActiveProposal(uint256 _proposalId) internal {
+    uint256 _snapshot = governorBravo.proposalSnapshot(_proposalId);
+    vm.roll(_snapshot + 1);
+
+    // Ensure the proposal is now Active
+    IGovernor.ProposalState _state = governorBravo.state(_proposalId);
+    assertEq(_state, IGovernor.ProposalState.Active);
+  }
+
+  function _jumpToVotingComplete(uint256 _proposalId) internal {
+    // Jump one block past the proposal voting deadline
+    uint256 _deadline = governorBravo.proposalDeadline(_proposalId);
+    vm.roll(_deadline + 1);
+  }
+
+  function _jumpPastProposalEta(uint256 _proposalId) internal {
+    uint256 _eta = governorBravo.proposalEta(_proposalId);
+    vm.roll(block.number + 1);
+    vm.warp(_eta + 1);
+  }
+
+  function _delegatesVoteOnBravoGovernor(uint256 _proposalId, uint8 _support) internal {
```

Comment:
nit: might want to change this to `_delegatesCastVoteOnBravoGovernor`. When I first saw it, I read it as delegates (verb) vote (noun), as in delegating a vote to someone, rather than delegates (noun) vote (verb) 😵‍💫

Reply:
Yes, good call out

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/13/commits/a44bd61cfde59a23f71cf0051f37ce4789a0a208

Reply:
Wrong commit, but I see the change is there 👍